src/sentry/models/deletedteam.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+from django.db import models
+
+from sentry.db.models import sane_repr, BoundedBigIntegerField
+from sentry.models.deletedentry import DeletedEntry
+
+
+class DeletedTeam(DeletedEntry):
+    """
+    This model tracks an intent to delete.
+    If an org is marked pending_delete through the UI, a deletedteam is created to log this deletion.
+
+    This model does not account for aborted or failed deletions
+    and is currently unable to log deletions that occur implicity
+    (i.e. when the sole parent object is deleted, the child is also marked for deletion as well).
```

Comment:
The wrapping on these comments is a little funny. I've seen most of our comments wrapped to column 80.